store/tools/ics23-iavl/cmd/testgen-iavl/main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	ics23 "github.com/confio/ics23/go"
+
+	iavlproofs "github.com/cosmos/cosmos-sdk/store/tools/ics23-iavl"
```

Comment:
this path is a bit odd, is there a reason for it? 

I think something like `github.com/cosmos/cosmos-sdk/store/tools/ics-23/iavl` may be better